packages/office-addin-usage-data-web/src/usageDataSettings.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as fs from "fs";
+import * as defaults from "./defaults";
+import { UsageDataLevel } from "./usageData";
+
+/**
+ * Allows developer to check if the program has already prompted before
+ * @param groupName Group name of the usage data object
+ * @returns Boolean of whether the program should prompt
+ */
+export function needToPromptForUsageData(groupName: string): boolean {
+    return !groupNameExists(groupName);
+}
+
+/**
+ * Reads data from the usage data json config file
+ * @returns Parsed object from json file if it exists
+ */
+export function readUsageDataJsonData(): any {

```

Comment:
seems like there might be some redundancy in all these read methods -we should figure out which ones we absolutely need and remove an redundancy